gdk4-x11/Cargo.toml

```
@@ -12,7 +12,8 @@ repository = "https://github.com/gtk-rs/gtk4-rs"
 version = "0.2.0"
 
 [features]
-dox = ["ffi/dox"]
+v4_4 = ["ffi/v4_4", "khronos-egl"]
+dox = ["ffi/dox", "v4_4"]
```

```suggestion
dox = ["ffi/dox"]
```
this doesn't make much sense, dox is a workaround around having to enable all the futures.

---

Without it, doc building fails:
```
error[E0432]: unresolved import `khronos_egl`
 --> gdk4-x11/src/x11_display.rs:6:5
  |
6 | use khronos_egl as egl;
  |     ^^^^^^^^^^^^^^^^^^ no external crate `khronos_egl`

error[E0432]: unresolved import `xkb`
 --> gdk4-wayland/src/wayland_device.rs:9:5
  |
9 | use xkb::Keymap;
  |     ^^^ use of undeclared crate or module `xkb`

error[E0432]: unresolved import `khronos_egl`
 --> gdk4-wayland/src/wayland_display.rs:6:5
  |
6 | use khronos_egl as egl;
  |     ^^^^^^^^^^^^^^^^^^ no external crate `khronos_egl`

error[E0432]: unresolved import `khronos_egl`
  --> gdk4-wayland/src/lib.rs:20:9
   |
20 | pub use khronos_egl;
   |         ^^^^^^^^^^^ no external crate `khronos_egl`

error[E0432]: unresolved import `xkb`
  --> gdk4-wayland/src/lib.rs:23:9
   |
23 | pub use xkb;
   |         ^^^ no external crate `xkb`
```

Should it exclude those related API when generating docs?

---

Yes, because you have the same issue on those as well. The `dox` feature should be dropped from imports

---

But the API depend on those crate.. I am lost. Please show me :)